toolkit/scripts/toolchain.mk

```diff
@@ -297,3 +307,26 @@ $(toolchain_rpms): $(TOOLCHAIN_MANIFEST) $(depend_REBUILD_TOOLCHAIN)
 		$(call print_error,\nToolchain download failed. See above errors for more details.) \
 	}
 endif
+
+# ./out/RPMS is reserved for RPMs generated by the tooling, all other RPMs are cached in the ./build folder. If REBUILD_TOOLCHAIN=y is set
+# the toolchain RPMs should also be placed into out/RPMs.
+# ./out/SRPMS are still handled by build_official_toolchain_rpms.sh as a side effect of building the toolchain tar.gz.
+ifeq ($(REBUILD_TOOLCHAIN),y)
+$(RPMS_DIR): $(toolchain_out_rpms)
+
+# For each toolchain rpm in ./out/RPMS, add a dependency on the counterparts in the normal toolchain directory:
+# Each path in $(toolchain_out_rpms) corresponds to a .rpm file we expect to have been built by the toolchain target and made available in ./out/RPMS.
+# Those rpms however are placed by default in ./build/toolchain/* (listed in $(toolchain_rpms)). So if we want a copy placed in ./out/RPMS 
+# we will need to copy it over. We can filter the list of toolchain rpms $(toolchain_rpms) to find the source that matches the target ($@),
+# then copy it over.
+$(toolchain_out_rpms): $(toolchain_rpms)
+	@src_rpm='$(filter %/$(notdir $@),$(toolchain_rpms))'  && \
+	if [ ! -f "$@" \
+			-o "$$src_rpm" -nt "$@" ] ; then \
+		echo "Placing built toolchain RPM $(notdir $@) into $(RPMS_DIR)" && \
+		cp $$src_rpm $@; \
```

Comment:
Are these two commands being printed in addition to the echo message itself? If so, I think we could silence them to reduce the noise and rely on the echo message itself. I already forgot how `make` handles this, so I apologize if I'm not making sense.:)

Reply:
The entire command is chained with `&& \`, the first line has a `@` which should suppress the rest of the output.